modules/ossm-troubleshooting-smcp.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-smcp_{context}"]
+= Troubleshooting the service mesh control plane
+
+If you are experiencing issues while deploying the service mesh control plane,
```

Comment:
```suggestion
If you are experiencing issues while deploying the Service Mesh control plane:
```
Update in the heading also, seems to use "Service Mesh control plane" throughout rest of docs in this PR so I'd keep the caps